recipes/ruamel.ordereddict/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: ruamel.ordereddict
+    version: 0.4.6
+
+build:
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    skip: True  # [py3k]
```

Comment:
Shame about this. See https://bitbucket.org/ruamel/ordereddict/issues/2/please-provide-a-python3-package.